.github/workflows/yamllinter.yaml

```diff
@@ -0,0 +1,27 @@
+name: Yaml Lint
+on:
+  pull_request:
+    paths:
+    - '**.yaml'
+
+jobs:
+  yaml-validator-and-linter :
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: yaml-lint
+      uses: ibiqlik/action-yamllint@v3
+        
+    - name: setup minikube
+      uses: manusa/actions-setup-minikube@v2.0.1
+      with:
+        minikube version: 'v1.13.1'
+        kubernetes version: 'v1.19.2'
+        start args: --memory 6g --cpus=2
+
+    - name: validate-yaml
+      run: |
+        cd cluster/examples/kubernetes/ceph/
```

Comment:
Since this requires minikube, how about if we move this step back to the canary test? We can keep this new action just for the yamllint tool.

Reply:
How about keeping this here till we fix the canary test because currently it's not stable or we can move to others?

Reply:
If we run this step before the unstable step (that waits for rook/csi pods), it will be fine. I don't want to wait for this check to be enabled until all the other linter issues are fixed either.

Reply:
Okay. Done